cinema/views.py

```diff
@@ -21,24 +28,52 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(mixins.ListModelMixin,
+                   mixins.CreateModelMixin,
+                   viewsets.GenericViewSet):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
 
+    def get_permissions(self):
+        if self.action == "create":
+            return [IsAdminUser()]
+        return [IsAuthenticated()]
```

Comment:
The `get_permissions` method in `GenreViewSet` restricts the `create` action to admins only. According to the task requirements, both list and create actions should be accessible to all users, not just admins. Consider removing the permission restriction for the `create` action.